src/js_realm.hpp

```diff
@@ -186,6 +185,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
     static void compact(ContextType, ObjectType, Arguments, ReturnValue &);
     static void delete_model(ContextType, ObjectType, Arguments, ReturnValue &);
     static void object_for_object_id(ContextType, ObjectType, Arguments, ReturnValue&);
+    static void privileges(ContextType, ObjectType, Arguments, ReturnValue&);
```

Comment:
Why no `#if REALM_ENABLE_SYNC`?

Reply:
The object store code takes care of abstracting away sync/non-sync logic (non-sync realms just always report that you have full permissions).